src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -48,38 +49,44 @@
                 abstract_arg('argument value resolvers'),
             ])
 
+        ->set('argument_resolver.attribute', AttributeValueResolver::class)
+            ->args([
+                tagged_locator('controller.argument_value_resolver', 'name'),
+            ])
+            ->tag('controller.argument_value_resolver', ['priority' => 150])
```

Comment:
I suggest using a priority of `1000` instead. This value resolver is really meant to be the first one.